<launch>
  <!-- set operating condition -->
  <param name="/operating_condition" type="int" value="0" />

  <!--launch the serial node-->
  <rosparam file="$(find serial_node)/data/serial_device_dict.xml" command="load"/>
  <node pkg="serial_node" type="multi_serial_topic" name="serial_node"
	output="screen" respawn="true" 
	launch-prefix="xterm -rv -e" />

  <!-- launch the wiimote control node -->
  <node pkg="puppeteer_control" type="new_wiimote"
	name="wii_controller" output="screen" respawn="true"
	launch-prefix="xterm -rv -e" />

  <!--launch the keyboard node-->
  <node pkg="keyboard_node" type="keyboard_node"
	name="keyboard_interface" output="screen" respawn="true"
	launch-prefix="xterm -rv -e" />

  <!-- launch the general wii node -->
  <node pkg="wiimote" type="wiimote_node.py"
	name="wiimote_connection" output="screen" respawn="true" />
</launch>

